### ISSUE TITLE
[Feature request] Progress indeterminate

### ISSUE THREAD

**Issue by guastallaigor** [growthbunker/vuedarkmode#29]
### What problem does this feature solve?

I did a few days ago the progress bar indeterminate for [PSone.css](https://98mprice.github.io/PSone.css/). If you think this is a good addition I can do it here aswell.

### What is your proposed solution?

A new prop (`indeterminate`) that would be a boolean (`default: false`) that when is true the progress add's an indeterminate animation that will ultimately ignore the progress prop.

**Comment by LeCoupa** [comment]
I'm just wondering: what are the situations when we would need to set indeterminate to true?

**Comment by guastallaigor** [comment]
I have only used once for a global progress bar on the top of the application, kind of like Nuxt.js has built in, but this one was indeterminate and it was visible in every request.

For the other parts of the application I've used spinners for the same effect, so not sure if this is a good addition or not.

I've seen other Vue.js frameworks having this, like [Vuesax](https://lusaxweb.github.io/vuesax/components/progress.html#indeterminate), [Vuetify](https://vuetifyjs.com/en/components/progress#example-linear-indeterminate), that's why I proposed this new feature.
